### DISCUSSION TITLE
测量组件label显示异常

### DISCUSSION THREAD

**Discussion by jiege15121313** [zouyaoji/vue-cesium#315] [General]
![1668414245233](https://user-images.githubusercontent.com/115357712/201614184-877081d0-89d9-481d-ad0b-c69daaed12f3.png)
![1668414303735](https://user-images.githubusercontent.com/115357712/201614198-cc7f1f7a-cf49-489d-8203-0458c2f654aa.png)
测量组件的labelOpts属性，绘制两个点可以正常显示，绘制两个以上边缘会出现虚化是什么问题呢？还有其他的label怎么不生效呢？

**Comment by zouyaoji** [comment]
请贴一下代码哈

**Comment by jiege15121313** [comment]
![6684725524045](https://user-images.githubusercontent.com/115357712/201797746-69b83bb0-8556-444c-8cd2-097fa909ea19.png)

**Comment by jiege15121313** [comment]
其他label没生效问题已经找到，原因是我只添加了labelOpts，没有添加labelsOpts，但是背景颜色虚化的问题还是存在呢
![16684750467128](https://user-images.githubusercontent.com/115357712/201802725-8b78fbfb-b8b6-4e8a-96bf-18396cc80b31.png)
